src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --accent-color: #00acdc;
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  padding: 10px 50px;
+  justify-content: space-between;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: black;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 13px 0;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+}
```

Comment:
not fixed
![image](https://github.com/mate-academy/layout_moyo-header/assets/70009205/946b35e2-de67-47e1-8609-f360a8b2db8f)